create_repo

```diff
@@ -34,29 +34,25 @@ if [[ $# != 1 ]]; then
     exit 0
 fi
 
-read -p "creer le depot '$1'? [Y/N] " answer
-if [ "`echo $answer | cut -c1`" != "Y" ]; then exit 1; fi
+read -p "Create Repository '$1' ? [y/n] " answer
+if [ "`echo $answer | cut -c1`" != "y" ]; then exit 1; fi
 
-
-token=`python2 -c "import hashlib; import getpass; print(hashlib.sha512((getpass.getpass().decode('utf8'))).hexdigest())"`
-
-blih.py -u $BLIH_USER -t $token repository create $1 || exit 1
-blih.py -u $BLIH_USER -t $token repository setacl $1 ramassage-tek r || exit 1
-blih.py -u $BLIH_USER -t $token repository setacl $1 rush r || exit 1
```

Comment:
Can you confirm to ma that there really is no way anymore to send a token to prevent from being promted the password every time the blih is called internally ?